test/failures.js

```diff
@@ -116,9 +116,51 @@ test('Invalid linting for larger code blocks read from fixtures', async (t) => {
     , 'message expected non-deprecated method'
     )
   })
+
+  t.test('function-call-argument-newline', async (t) => {
+    const [result] = await linter.lintFiles(['function-call-argument-newline-fixture'])
+    const messages = result.messages
+    t.equal(result.errorCount, 3, 'error count')
+
+    t.equal(
+      messages[0].message
+    , 'Expected newline after \'(\'.'
+    , 'incorrect function param newlines'
+    )
+
+    t.equal(
+      messages[1].message
+    , 'Expected newline between arguments/params.'
+    , 'incorrect function param newlines'
+    )
+
+    t.equal(
+      messages[2].message
+    , 'There should be a line break after this argument.'
+    , 'incorrect function param newlines'
```

Comment:
With all these descriptions the same, it's a lot of work to trace back to the offending code, especially since there's only 2 calls but 3 errors.  Can we be more explicit with something like `func call 2 params should be on separate lines` etc etc?

Reply:
Good call. Updated comments here as well as for the `function-paren-newline` failure tests.